lightwood/api/types.py

```diff
@@ -228,6 +231,8 @@ class TimeseriesSettings:
         # @TODO: George: No, I don't think it is, we need to pass this some other way
     )
     allow_incomplete_history: bool = False
+    eval_cold_start: bool = True
```

Comment:
this will absolutely require a tutorial example, as it's a pretty "advanced" feature